code/modules/food/cooking_machines.dm

```diff
@@ -478,58 +478,58 @@ var/global/ingredientLimit = 10
 	empty_icon()
 
 
-// Sugarizer ///////////////////////////////////////
+// confectionator ///////////////////////////////////////
 // its like a deepfrier
 
 // but with sugar
 
-#define SUGARIZER_MINSUGAR 50
+#define confectionator_MINSUGAR 50
 
-/obj/machinery/cooking/deepfryer/sugarizer
-	name = "sugarizer"
+/obj/machinery/cooking/deepfryer/confectionator
```

Comment:
You need to give this `machine_flags = WRENCHANCHOR | CROWDESTROY | SCREWTOGGLE`